framework/secrets.py

```diff
@@ -43,6 +43,7 @@ class Secrets(ndb.Model):
   session_secret = ndb.StringProperty()
 
   @classmethod
+  @ndb.transactional(retries=4)
```

Comment:
This is still needed. More details in https://github.com/GoogleChrome/chromium-dashboard/pull/3184#issuecomment-1697546461

Reply:
Ah, so there is basically a race to create a new session secret in the case where there was none previously (which is a case that happens on every test run, even though developers only see it once when setting up their development environment).  I think this race condition exists on GAE since we migrated to python3 (and gunicorn).  We didn't encounter this problem because the session secret already existed from running in python2.